content/en/01-about-pixie/05-faq.md

```diff
@@ -5,7 +5,7 @@ metaDescription: "Commonly asked questions about Pixie."
 order: 5
 ---
 
-### General
+**General**
```

Comment:
I needed to make this body text (and not H3) because there's a weird bug with our `docs.px.dev` website where it doesn't like the fact that these H3 don't have an H2 parent. h-links for H3 are of the format #-H2-H3 and these links get random H2 parents that can be brokenlinks.